app/javascript/app/selectors/climate-policies-selectors.js

```diff
@@ -12,7 +15,21 @@ export const policiesListBySector = createSelector(
   }
 );
 
+export const policiesByCode = createSelector(getClimatePoliciesList, list => {
+  if (!list) return null;
+  const groupedByCode = groupBy(list, 'code');
+  const policies = {};
+  Object.keys(groupedByCode).forEach(code => {
```

Comment:
Just a matter of taste, but wouldn't it be cleaner avoiding the creation of `policies` and `policies[code]` empty objects using `reduce` instead of `forEach`?
```js
export const policiesByCode = createSelector(getClimatePoliciesList, list => {
  if (!list) return null;
  const groupedByCode = groupBy(list, 'code');
  return Object.keys(groupedByCode).reduce((acc, code) => ( { ...acc, [code]: groupedByCode[code][0] }), {})
});
```

Reply:
yes, totally! thanks for pointing it out